torch/_torch_docs.py

```diff
@@ -5839,7 +5839,7 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.where,
            r"""
-where(condition, x, y) -> Tensor
+.. function:: where(condition, x, y) -> Tensor
```

Comment:
isn't this either a different function or the optional `x, y` need to be explained in some way?

Reply:
whoops, didn't see the line below, my mistake.